src/lib/components/Login/Login.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Cookie from 'js-cookie';
+import { ONA, SupAuth } from 'gisida';
+import './Login.scss';
+import { Router } from '../../routes/router';
+
+const { defaultUnSupAuthZ: deAuthZ } = SupAuth;
+
+const mapStateToProps = (state) => {
+  const { APP } = state;
+  return {
+    appPassword: APP.password,
+    global: state,
+    appIcon: APP.appIcon,
+    loginIcon: APP.loginIcon,
+    publicUsername: APP.publicUsername,
+    publicPassword: APP.publicPassword
+  };
+};
+
+export const isLoggedIn = function() {
+  return Cookie.get('dsauth') === 'true';
+};
+
+export const logOut = (e) => {
+  e.preventDefault();
+  deAuthZ();
+  Cookie.set('dsauth', false);
+  Router.history.push('/login');
+};
+
+class Login extends Component {
+  constructor(props) {
+    super(props);
+    const clientID = 'CdJqBZYRVrbpnAu4JoYYFXFPQJa3xWi25oDPqnRY';
```

Comment:
@ciremusyoka Was this line copied from somewhere? (seems like it was)

1. **This should never happen**, _especially in public repositories_, please move into an `.env` file which does not get checked into git history
2. This should be provided by the client repository (from an env file) and passed into the component as a prop, though that's not necessary for this PR since we need to iterate upon this to be integrated with Gatekeeper

Reply:
This is a security vulnerability and needs to be resolved (outside of this PR), I will create a ticket.

Reply:
No ticket yet, but has been escalated https://onaio.slack.com/archives/C0FTTFHB2/p1580492134007900